packages/bot-web-ui/src/stores/root-store.ts

```diff
@@ -19,10 +19,42 @@ import SelfExclusionStore from './self-exclusion-store';
 import ToolboxStore from './toolbox-store';
 import AppStore from './app-store';
 import DashboardStore from './dashboard-store';
+import { TDbot, TWebSocket } from 'Types';
+import { TStores } from '@deriv/stores';
```

Comment:
@shafin-deriv Please update your branch with the latest master, This should be `TCoreStores` instead of `TStores` 🙇🏻

Reply:
Then use `TRootStores` instead 😅 It's the same thing but in master we have renamed it to `TCoreStores` so after you guys update the branch you can change the name 🙇🏻